internal/sdkprovider/service/serviceintegration/service_integration.go

```diff
@@ -292,8 +282,8 @@ func resourceServiceIntegrationWaitUntilActive(ctx context.Context, d *schema.Re
 				return nil, notActive, nil
 			}
 
-			if ii.IntegrationType == "kafka_connect" && ii.DestinationService != nil {
-				if _, err := client.KafkaConnectors.List(ctx, projectName, *ii.DestinationService); err != nil {
+			if ii.IntegrationType == "kafka_connect" && ii.DestService != nil {
+				if _, err := client.ServiceKafkaConnectList(ctx, projectName, *ii.DestService); err != nil {
```

Comment:
The operation id says "kafka connect", though it is for the kafka connector https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceKafkaConnectList
```
https://api.aiven.io/v1/project/{project}/service/{service_name}/connectors
```